podium-uaa/src/main/resources/i18n/messages.properties

```diff
@@ -46,3 +46,5 @@ email.signature.servicelink=Selfservice: http://trait.topdesk.net
 email.signature.telephone=Telefoon: +31 (0)881167500
 
 email.autogenerated=This is an automatically generated message. If you have any questions, please send an email to servicedesk@ctmm-trait.nl
+
+podium.ValidPassword.message=The password is invalid.
```

Comment:
"Password is correct" / "Password is incorrect" seems to be the right feedback for login, "Password is invalid" seems to be right feedback for an error when updating a password.